src/main.cpp

```diff
@@ -33,9 +33,11 @@ using namespace adiscope;
 
 int main(int argc, char **argv)
 {
+#ifdef Q_OS_WIN
 	/* Set environment variable QT_AUTO_SCREEN_SCALE_FACTOR to 1(true) thus
 	 * making Scopy aware of the scaling settings set by the OS */
 	qputenv("QT_AUTO_SCREEN_SCALE_FACTOR", QString("1").toLocal8Bit());
+#endif
```

Comment:
a commit description would be useful about this; and why it's being done on Linux;

Reply:
Forgot to add the description. This was initially just a test commit to see if the issue would be fixed. Thank you for noticing the missing description.